packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hooks best practices
+excerpt: >-
+  Understand the best practices to implement and secure Okta Event Hooks or Inline Hooks.
+---
+
+# Hooks best practices
+
+Event Hooks and Inline Hooks are outbound calls from Okta to an external service. These outbound calls integrate custom functionality into your Okta processes. See [Event Hooks](/docs/concepts/event-hooks) and [Inline Hooks](/docs/concepts/inline-hooks) for additional concept information.
+
+The following sections review best practices to implement and secure Okta Event Hooks or Inline Hooks.
+
+## Secure your Hook endpoint
+
+To prevent a malicious actor from making requests to the endpoint where your Okta Hooks are sent, use the following best practices:
+
+* Configure Okta to send an authentication header in the Hook and validate it in every request by one of two ways:
+
+  * Using [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication). When activating and enabling your Hooks on the Okta org, set the **Authorization field** as `authorization` and the **Authentication secret** in the Base64 `user:password` format.
+
+    >**Note:** You must include the authentication scheme as part of the **Authentication secret**. For Basic Authentication, your secret must appear similar to: `Basic Base64(user:password)`. See the following partial Hook header as an example:
+
+    ```cURL
+    'content-type',
+    'application/json',
+    'authorization',
+    'Basic YWRtaW46c3VwZXJzZWNyZXQ='
+    ```
```

Comment:
The formatting of this header is not very readable - can we use the same formatting used in this example:

https://developer.okta.com/docs/concepts/event-hooks/#http-headers